modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -42,6 +43,18 @@ master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
 worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
 ----
 
+. Change to the `eus-4.10` channel, run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.10
+----
+
+[NOTE] If receive an error message indicating that eus-4.10 is not one of the
```

Comment:
Note style is as follows, and I made some changes to the text:

```
[NOTE]
====
If you receive an error message indicating that `eus-4.10` is not one of the 
available channels this indicates that Red Hat is still rolling out 4.8 to 4.10 EUS upgrades. 
This rollout process generally takes 45-90 days starting at the GA date.
====
```

Reply:
Thanks, updated that.